### PR TITLE
SQL utils package

### PR DESCRIPTION
**Pull request by lewish**
Publishes the classes under sql/ as a vanilla package that can be used directly in dataform projects, or non dataform code.
Replaces crossdb, as it's not very useful and moves it's one timestamp method into the sql class (And crossdb was only used in a few of our integration tests).

### CODE REVIEW

File: packages/@dataform/sql/BUILD [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+)
+
+pkg_bundle_dts(
+    name = "bundle.d",
+    entry_point = "index.ts",
+    externals = [],
+    deps = [
+        ":sql",
+    ],
+)
```
Comment (Ekrekr) [inline packages/@dataform/sql/BUILD:45]: Nice

File: sql/index.ts [hunk 2]
```diff
@@ -142,6 +142,19 @@ export class Sql {
     return `unix_millis(${timestamp})`;
   }
 
+  public timestampCurrentUTC() {
```
Comment (BenBirt) [inline sql/index.ts:145]:
```suggestion
  public currentUtcTimestamp() {
```

**Comment by lewish** [comment]
> I'm slightly uncomfortable that this means we now need to (major-version) bump `version.bzl` for every API change in `sql` _or_ we now are making the promise not to make any backwards-incompatible changes to `sql` until we want to major-version bump `@dataform/core`.
> 
> Can we version `sql` separately?

Done!

File: sql/index.ts (hunk 2 above)
Comment (lewish) [reply sql/index.ts]: Have moved all timestamp functions into a `timestamps` property and new class which makes them nicer to read.

File: tests/integration/sqldatawarehouse_project/definitions/example_incremental.sqlx [hunk 3]
```diff
@@ -3,11 +3,11 @@ config {
...
 
 js {
-  const crossdb = require("@dataform/crossdb");
+  const sql = require("@dataform/sql");
 }
 
 ${ when(incremental(), "select * from (") }
 
-select ${crossdb.currentTimestampUTC()} as ts
+select ${sql().timestamps.currentUTC()} as ts
```
Comment (BenBirt) [inline tests/integration/sqldatawarehouse_project/definitions/example_incremental.sqlx:11]:
```suggestion
select ${sql("mssql").timestamps.currentUTC()} as ts
```

**Pull request merged**